lib/Gateway.js

```diff
@@ -1528,7 +1530,7 @@ Gateway.prototype.discoverValue = function (node, vId) {
         if (valueId.list) {
           cfg.discovery_payload.value_template = getMappedStateTemplate(
             valueId.states,
-            0
+            valueId.default ? valueId.default : 0
```

Comment:
Just to be sure I understood this correctly, IMO when the value is not in the map the value shown should be the one coming from json payload. Id this what's happening here?

Reply:
I stand corrected. I did some nice patching on the payload, after manual testing

If `valueId.default` is undefinied the default value of payload is the `value_json.value`
this way it really presents the value numeric or other on UI